Rakefile

```
@@ -1,6 +1,2 @@
-# Add your own tasks in files placed in lib/tasks ending in .rake,
-# for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
-
-require_relative 'config/application'
-
+require_relative "config/application"
```

```suggestion
require_relative "config/application"
```
require_relative 'config/application'